commands.js

```diff
@@ -2007,6 +2007,7 @@ exports.commands = {
 	blacklist: function (target, room, user) {
 		if (!target) return this.parse('/help blacklist');
 		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (target === 'show') return this.parse('/showblacklist');
```

Comment:
Maybe there should be a way to group together multiple commands so showing the help for one shows the help for all of them...